cmd/all-in-one/all_in_one_test.go

```diff
@@ -138,6 +142,18 @@ func healthCheck() error {
 	return fmt.Errorf("query service is not ready")
 }
 
+func faviconCheck() error {
```

Comment:
I don't think this one is testing the docker image

Reply:
Hi Yuri,
My understanding may be wrong but looking at this code https://github.com/jaegertracing/jaeger/blob/main/scripts/build-all-in-one-image.sh#L23-L32 it seems we first bring up the newly built all-in-one image and then run the integration test from this file against the container.